frontend/ui/widget/focusmanager.lua

```diff
@@ -319,7 +321,7 @@ function FocusManager:moveFocusTo(x, y, focus_flags)
         self.selected.x = x
         self.selected.y = y
         -- widget create new layout on update, previous may be removed from new layout.
-        if Device:hasDPad() then
+        if focus_flags == FocusManager.FORCED_FOCUS or Device:hasDPad() then
```

Comment:
`if bit.band(focus_flags, FocusManager.FORCED_FOCUS) == FocusManager.FORCED_FOCUS`

Reply:
What is the reason of this complication that gives the same result?

Reply:
`focus_flags` is a bitfield, any and all of the flags can be OR'ed in it, it's not 1:1.

I'm not entirely sure anything actually *relies* on that right now, but that's how it was designed ;o).